command/v7/copy_source_command.go

```diff
@@ -48,6 +49,14 @@ func (cmd *CopySourceCommand) ValidateFlags() error {
 		}
 	}
 
+	if cmd.Strategy.Name == constant.DeploymentStrategyDefault && cmd.MaxInFlight != nil {
```

Comment:
[suggestion, blocking] I believe this is now incorrect: MaxInFlight can take a value of 0, which should be invalid for this flag

Reply:
LGTM